bpf/lib/common.h

```diff
@@ -572,7 +572,7 @@ struct ct_entry {
 struct lb6_key {
 	union v6addr address;	/* Service virtual IPv6 address */
 	__be16 dport;		/* L4 port filter, if unset, all ports apply */
-	__u16 slave;		/* Backend iterator, 0 indicates the master service */
+	__u16 endpoint;		/* Backend iterator, 0 indicates the primary service */
```

Comment:
I note that here we call it `endpoint` (which admittedly is consistent with k8s terminology, but also collides with long-held Cilium terminology for the container network termination point), yet in the comment we clarify it by calling it the backend.

Did you consider frontend/backend terminology?

Reply:
frontend/backend would actually be more clear, or, short `fe`/`be` if vars become too long.

Reply:
yeah I thought about the same the first time I saw this PR but I couldn't think on an alternative, `fe/be` seems a good approach to me as well.

Reply:
Yep, frontend/backend makes sense. However, the particular field at the line 575 points to a "service slot" (`lb6_service`) which contains a backend_id pointing to a real backend. This is the main reason why I didn't want to use `backend' here. Maybe we should rename `slave` to `backend_slot` or `backend_no`?

Reply:
if it contains a `backend_id`, is there anything stopping you from calling it `backend_id`? :-)

id/slot/no, any of these sound reasonable.

Reply:
Unfortunately, it doesn't contain an ID: `lb6_key{.slave = 1 ... lb6_service[lb6_key{.slave = 0}].count} => lb6_service.backend_id => lb6_backend`. As you can see, `slave` refers to a "backend slot index" ;-) OK, I think I'm going to rename to `backend_slot`, and add a short comment about relations.